### PR TITLE
[DEV-2435]&[DEV-2344]&[DEV-2425]&[DEV-2427] Static Data Caching, Swap amlCheck + Refactoring, aml auto result

### PR DESCRIPTION
**Pull request by Yannick1712**
#### Pre-Release

- [x] Check migrations
  - No database related infos (sqldb-xxx)
  - Impact on GS (new/removed columns)
- [x] Check for linter errors (in PR)
- [x] Test basic user operations (on [DFX services](https://dev.services.dfx.swiss))
  - Login/logout
  - Buy/sell payment request
  - KYC page

#### Post-Release

- Test basic user operations
- Monitor application insights log

### CODE REVIEW

File: src/subdomains/core/aml/aml-helper.service.ts [hunk 1]
```diff
@@ -125,6 +123,16 @@ export class AmlHelperService {
         )
           errors.push(AmlError.CARD_BLACKLISTED);
         if (last7dVolume > Config.tradingLimits.weeklyAmlRule) errors.push(AmlError.WEEKLY_LIMIT_REACHED);
+      } else {
+        // swap
+        if (
+          last24hVolume > Config.tradingLimits.dailyDefault &&
+          (entity.userData.kycLevel < KycLevel.LEVEL_50 || !entity.userData.hasBankTxVerification)
+        ) {
+          errors.push([AmlError.DEPOSIT_LIMIT_REACHED, AmlError.KYC_LEVEL_TOO_LOW]);
```
Comment (davidleomay) [inline src/subdomains/core/aml/aml-helper.service.ts:132]: This check is already there (lines 46 + 47) and can be removed IMO

File: src/subdomains/supporting/bank/bank-account/bank-account-admin.controller.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { BankAccount } from './bank-account.entity';
 import { BankAccountService } from './bank-account.service';
 import { CreateBankAccountDto } from './dto/create-bank-account.dto';
 
-@ApiTags('BankAccount')
+@ApiTags('Bank Account')
```
Comment (Yannick1712) [inline src/subdomains/supporting/bank/bank-account/bank-account-admin.controller.ts:10]: `Bank-Account` or `Bank account` ?

File: src/subdomains/supporting/bank/bank-account/bank-account.controller.ts [hunk 3]
```diff
@@ -14,7 +14,7 @@ import { CreateBankAccountDto } from './dto/create-bank-account.dto';
 import { CreateIbanDto, IbanDto } from './dto/iban.dto';
 import { UpdateBankAccountDto } from './dto/update-bank-account.dto';
 
-@ApiTags('BankAccount')
+@ApiTags('Bank Account')
```
Comment (Yannick1712) [inline src/subdomains/supporting/bank/bank-account/bank-account.controller.ts:17]: `Bank-Account` or `Bank account` ?

File: src/subdomains/supporting/bank/bank-account/bank-account-admin.controller.ts (hunk 2 above)
Comment (davidleomay) [reply src/subdomains/supporting/bank/bank-account/bank-account-admin.controller.ts:10]: image
ApiTags are shown as titles in the Swagger docu and titles are written in capital letters 😉

**Review by davidleomay — APPROVED** [review]

**Pull request merged**